.circleci/config.yml

```diff
@@ -83,7 +91,16 @@ jobs:
           command: poetry check
           name: dependencies
       - run:
-          command: poetry run flake8 --ignore=E501,E402 .
+          command: |
+            sudo apt-get install jq
+            if COMMAND_OUTPUT=$(poetry check && poetry run flake8 --ignore=E501,E402 . 2>&1); then
```

Comment:
I can't tell if this if statement is checking something about COMMAND_OUTPUT (that is, if it's like C where an assignment returns the value stored) or if it's checking the return value of the poetry command...

I think it would be better to move the assignment outside the if statement; and have the if statement either check COMMAND_OUTPUT, or store the return value with something like COMMAND_RETURN=$? and then check the return value.

In general, more explicit for people who can't read bash as well as you ;)

Reply:
`$?` after the poetry command and variable assignment ends up being the return code for `poetry run ...` but it being run in the if statement flips that (to be honest I can't really tell why) so true we just echo the log and pass then false we report the failure and send out an `exit 1` for the CI dashboard.

The reason I placed the command in the if clause was to mimic a try/catch block and stop the command from immediately halting the CI task without reporting to the github pr. I'm not sure what would be better here aside from maybe a comment :/